test/ledger.integration.test.js

```diff
@@ -391,18 +454,262 @@ test('integration : v2 grant contribution workflow with uphold BAT wallet', asyn
 
   viewingCredential.finalize(response.body.verification)
 
-  const votes = ['wikipedia.org', 'reddit.com', 'youtube.com', 'ycombinator.com', 'google.com']
+  const votes = ['wikipedia.org', 'reddit.com', 'youtube.com', 'ycombinator.com', 'google.com', publisher]
   // const votes = ['basicattentiontoken.org']
   for (var i = 0; i < surveyorIds.length; i++) {
     const id = surveyorIds[i]
+    let publisher = votes[i % votes.length]
     response = await request(srv.listener)
       .get('/v2/surveyor/voting/' + encodeURIComponent(id) + '/' + viewingCredential.parameters.userId)
       .expect(ok)
 
     const surveyor = new anonize.Surveyor(response.body)
     response = await request(srv.listener)
       .put('/v2/surveyor/voting/' + encodeURIComponent(id))
-      .send({'proof': viewingCredential.submit(surveyor, { publisher: votes[i % votes.length] })})
+      .send({'proof': viewingCredential.submit(surveyor, { publisher })})
       .expect(ok)
   }
 })
+// wipe owner and publisher
+// send votes,
+// pull contributions report
+
+//
+// const quote = {
+//   '"': 1,
+//   "'": 2
+// }
+test('get contribution data', async t => {
+  t.plan(1)
+  const { BAT_EYESHADE_SERVER: domain } = process.env
+  const url = formURL('/v1/reports/publishers/contributions')
+  const res = await req({ url, domain })
+  const { body: bod } = res
+  const { reportId } = bod
+  const res2 = await fetchReport({
+    domain,
+    reportId
+  })
+  const {
+    // text: body,
+    status
+  } = res2
+  t.is(status, 200)
+  // const json = body.split('\n').map(row => row.split(',').map(cell => {
+  //   const last = cell.length - 1
+  //   const first = quote[cell[0]]
+  //   if (first && first === quote[cell[last]]) {
+  //     return cell.slice(1, last)
+  //   } else {
+  //     return cell
+  //   }
+  // }))
+  // console.log('contribution data', reportId, json)
```

Comment:
we're not checking that the contribution report is correct here right? might be a good TODO